### PR TITLE
Copy of #2362

### PR DESCRIPTION
**Pull request by jinroq**
Copy of #2362 .
@hachi8833

Revised the points pointed out in the review.

### CODE REVIEW

File: ja/news/_posts/2019-12-12-separation-of-positional-and-keyword-arguments-in-ruby-3-0.md [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+
+def foo(*args, **kwargs, &block)
+  target(*args, **kwargs, &block)
+end
+
+foo() #=> Ruby 2.6以前: [{}]
+      #=> Ruby 2.7以降:  []
+{% endhighlight %}
+
+`foo()`には引数がありませんが、Ruby 2.6では空のハッシュ引数が`target`に渡されます。理由は、メソッド`foo`が明示的にキーワード（`**kwargs`）を委譲しているためです。`foo()`が呼び出されると、`args`は空のArrayになり、`kwargs`は空のHashになり、blockはnilになります。。そして`target(*args, **kwargs, &block)`は空のHashを引数として1つ渡します。理由は、`**kwargs`が自動的にHash位置引数に変換されるためです。
```
Comment (mame) [inline ja/news/_posts/2019-12-12-separation-of-positional-and-keyword-arguments-in-ruby-3-0.md:350]:
```suggestion
`foo()`には引数がありませんが、Ruby 2.6では空のハッシュ引数が`target`に渡されます。理由は、メソッド`foo`が明示的にキーワード（`**kwargs`）を委譲しているためです。`foo()`が呼び出されると、`args`は空のArrayになり、`kwargs`は空のHashになり、`block`は`nil`になります。そして`target(*args, **kwargs, &block)`は空のHashを引数として1つ渡します。理由は、`**kwargs`が自動的にHash位置引数に変換されるためです。
```

**Comment by mame** [comment]
Thank you! Could you please rebase the pull request? Then I'll merge it soon.

**Comment by jinroq** [comment]
@mame I did it.

**Comment by mame** [comment]
Thanks!

**Pull request merged**